Source/Library/Adapt.Analytics/NominalFrequencyAnalytic.cs

```diff
@@ -0,0 +1,98 @@
+﻿// ******************************************************************************************************
+//  PassThrough.tsx - Gbtc
+//
+//  Copyright © 2021, Grid Protection Alliance.  All Rights Reserved.
+//
+//  Licensed to the Grid Protection Alliance (GPA) under one or more contributor license agreements. See
+//  the NOTICE file distributed with this work for additional information regarding copyright ownership.
+//  The GPA licenses this file to you under the MIT License (MIT), the "License"; you may not use this
+//  file except in compliance with the License. You may obtain a copy of the License at:
+//
+//      http://opensource.org/licenses/MIT
+//
+//  Unless agreed to in writing, the subject software distributed under the License is distributed on an
+//  "AS-IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. Refer to the
+//  License for the specific language governing permissions and limitations.
+//
+//  Code Modification History:
+//  ----------------------------------------------------------------------------------------------------
+//  08/02/2021 - C. Lackner
+//       Generated original version of source code.
+//  12/16/2021 - A. Hagemeyer
+//       Changed to a Nominal Frequency Analytic
+//
+// ******************************************************************************************************
+
+
+using Adapt.Models;
+using GemstoneCommon;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Adapt.DataSources
+{
+    /// <summary>
+    /// if the frequency value/base frequency is between the max and min values, return frequency value. Otherwise return NaN
+    /// </summary>
+    
+    [AnalyticSection(AnalyticSection.DataCleanup)]
+    [Description("Nominal Frequency: Returns frequency if frequency / baseFrequency is between max and min.")]
+    public class NominalFrequency: IAnalytic
+    {
+        private Setting m_settings;
+        private int m_fps;
+        public class Setting
+        {
+            public double BaseFrequency { get; set; }
```

Comment:
instead of `BaseFrequency` it should be called `NominalFrequency` also consider attaching an attribute for spacing in the UI